src/main/docker/Dockerfile.jvm

```diff
@@ -15,7 +15,7 @@
 #
 ###
 FROM fabric8/java-alpine-openjdk8-jre:1.6.5
-ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
+ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Dconfig.secret.path=/run/secrets/git-eca-rest-api/secret.properties"
```

Comment:
I'm not found of hardcoding it in the docker image itself. I would rather add inject environment variable in the pod's container (see https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/#using-environment-variables-inside-of-your-config). Of course, this will override the `JAVA_OPTIONS` env from the docker image, so you will need to repeat what's in there by default (ie `-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager`)